rasa/nlu/components.py

```diff
@@ -88,11 +88,31 @@ def validate_required_components_from_data(
 ):
 
     response_selector_exists = False
+    regex_config_exists = False
+    crfentity_config_exists = False
+
     for component in pipeline:
+        if "RegexFeaturizer" in component.name:
+            regex_config_exists = True
+        if "CRFEntityExtractor" in component.name:
+            crfentity_config_exists = True
```

Comment:
could do something like 
```
regex_config_exists = any([component.name = "RegexFeaturizer" for component in pipeline])
```

or even 
```
if data.regex_features and not any([component.name = "RegexFeaturizer" for component in pipeline]) 
```
in line 104

Reply:
I'll look at this but that 2nd option there gives invalid syntax so I'd have to see what it doesn't like

Reply:
ok I implemented this code for them I like it a lot better as well

Reply:
ah yes ofc haha